cubitpy/cubitpy.py

```diff
@@ -243,6 +242,7 @@ def add_element_type(
         # Add data that will be written to bc file.
         self.blocks.append([el_type, " ".join([material, bc_description])])
 
+
```

Comment:
Seems like this does not align to the `black` formatter. (Also the change above?)

Reply:
I forgot to apply the `black` formatter.  Is this somewhere documented?